pkgs/applications/audio/pbpctrl/default.nix

```
@@ -0,0 +1,35 @@
+{ rustPlatform
+, fetchFromGitHub
+, pkg-config
+, dbus
+, protobuf
+, lib
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "pbpctrl";
+  version = "v0.1.1-2";
+
+  # https://github.com/qzed/pbpctrl/issues/4
+  src = fetchFromGitHub {
+    owner = "qzed";
+    repo = "${pname}";
+    rev = "9fef4bb88046a9f00719b189f8e378c8dbdb8ee6";
+    hash = "sha256-8YbsBqqITJ9bKzbGX6d/CSBb8wzr6bDzy8vsyntL1CA=";
+  };
+
+  cargoHash = "sha256-sXbae1XfIiyvW2JaoHzI4vImOGBenZcZuVvJlYwdvQ8=";
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ dbus.dev ];
+
+  PROTOC = "${protobuf}/bin/protoc";
```

```suggestion
  PROTOC = "protoc";
```
Or just delete it of not required.